tests/queries/0_stateless/02864_statistics_uniq.reference

```diff
@@ -16,7 +16,7 @@ After merge
 After modify TDigest
         Prewhere info
           Prewhere filter
-          Prewhere filter column: and(less(a, 10), equals(c, 11), less(b, 10)) (removed)
+          Prewhere filter column: and(equals(c, 11), less(a, 10), less(b, 10)) (removed)
```

Comment:
Emmmm.... didn't know why the result of this unit test still changes.

Reply:
Well it's because we still keep https://github.com/ClickHouse/ClickHouse/pull/65792/files#diff-19ac6698d009b3c713a07501cec76122bf6b4828bb4e585f36b84d86aeab17f9R103